AndroidNativeLoginTemplate/app/src/main/java/com/salesforce/androidnativelogintemplate/NativeLogin.kt

```diff
@@ -35,7 +35,6 @@ import android.widget.Toast
 import android.widget.Toast.LENGTH_LONG
 import android.window.OnBackInvokedDispatcher.PRIORITY_DEFAULT
 import androidx.activity.ComponentActivity
-import androidx.activity.enableEdgeToEdge
```

Comment:
NIT:  As mentioned before, [apps targeting Android 15 will be edge to edge by default](https://developer.android.com/about/versions/15/behavior-changes-15#edge-to-edge).